package-lock.json

```diff
@@ -11447,16 +12170,17 @@
       }
     },
     "node_modules/fqm-execution": {
-      "version": "1.6.0",
-      "resolved": "https://registry.npmjs.org/fqm-execution/-/fqm-execution-1.6.0.tgz",
-      "integrity": "sha512-z7aejBvjENRs7AX8egmvalwnMuXcU0maladFpqVsltzZGBPE72H8pPJzOT8fzFhJUdkYQ1Wo3PdoHeg92aW4/Q==",
+      "version": "1.7.0",
+      "resolved": "https://registry.npmjs.org/fqm-execution/-/fqm-execution-1.7.0.tgz",
+      "integrity": "sha512-XMpCLfysi36QYB7jIQ3ZyJmME4R4LD6+hOiKRI3uxgqZcBP5oJAqzd0K6+7m76kJ0J2kjkTNtEr+4Gh3DfG3zg==",
       "license": "Apache-2.0",
       "dependencies": {
         "@types/fhir": "0.0.34",
         "axios": "^0.21.1",
         "commander": "^6.1.0",
         "cql-exec-fhir": "^2.1.3",
-        "cql-execution": "^3.1.0",
+        "cql-execution": "^3.2.0",
```

Comment:
not something under our control exactly, but we might want to afterparty this... not sure this should be merged before next release is tagged as new cql-execution would need heavier QA testing